09a_Custom_Coadd.ipynb

```diff
@@ -0,0 +1,1386 @@
+    "\n",
+    "Use it to retrieve the newly made Coadd (which is named deepCoadd by default) from the simple butler.\n",
+    "\n",
+    "So long as you use the simpleButler, it will search the collections in order, and since we made sure that the new collection we just created and wrote to appears first -- as checked in the first cell below -- it will take the deepCoadd from that new collection."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "15c45e5b-9c01-4d9a-acf1-8780ec93b5de",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# simpleButler.registry.getCollectionChain(my_outputCollection)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "74e56269-f945-4a69-9931-4b0b63cda199",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# my_new_deepCoadd = simpleButler.get('deepCoadd', dataId=my_dataId)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "86f2e113-1cd4-4dbd-9d91-7e4075417f5c",
+   "metadata": {},
+   "source": [
+    "#### Accessing your run results in another notebook.\n",
+    "\n",
+    "The results have been permanently written to the butler.\n",
+    "\n",
+    "If you instantiated a new butler in a new notebook: <br>\n",
+    "`config = \"dp02\"`<br>\n",
+    "`collection = \"u/[Your User Name]/[Today's Date]/coadd_recreation_nb/TestWindow1_coadd\"`<br>\n",
+    "`butler = Butler(config, collections=collection)`<br>\n",
+    "\n",
+    "Then you would be able to define the dataId in the same way:<br>\n",
+    "`my_dataId = {'band': 'i', 'tract': 4431, 'patch': 17}`<br>\n",
+    "\n",
+    "And retrieve your new deepCoadd with:<br>\n",
+    "`my_new_deepCoadd = Butler.get('deepCoadd', dataId=my_dataId)`.\n",
+    "\n",
+    "Since we already have our collection in the `simpleButler`, we don't need this option in this notebook.\n",
+    "\n",
+    "This will be demonstrated in tutorial notebook 09b_Custom_Coadd_Sources.ipynb."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "40f23100-15af-4e0c-b036-d507eeaffe36",
+   "metadata": {},
+   "source": [
+    "### 4.2.3. Check the inputs of the new Coadd\n",
+    "\n",
+    "Check the inputs of my_new_deepCoadd, and compare them with the \"queryString\", and see that they match."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "42a2d904-2c31-4f48-b564-a122978371f4",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "my_new_deepCoadd_inputs = simpleButler.get(\"deepCoadd.coaddInputs\", my_dataId)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "971b923c-dc89-41c1-8e79-9b30d2152343",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "my_new_deepCoadd_inputs.visits.asAstropy()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a42df602-9d76-4419-9fef-25bd3fc9e874",
+   "metadata": {},
+   "source": [
+    "### 4.2.4. Display the new Coadd"
```

Comment:
When displaying the new coadd, it might be informative to display a zoomed-in portion of it next to the `deepCoadd` containing _all_ 161 i-band images. And, even better would be to show 3 panels with a single calexp, the 6-image coadd, and the 161-image coadd, to illustrate the (hopefully obvious) differences. Just a thought...

Reply:
That happens in NB 09b. Added note that this happens in 09b.